pkg/occlient/occlient_test.go

```diff
@@ -8,9 +8,45 @@ import (
 	"testing"
 
 	routev1 "github.com/openshift/api/route/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	ktesting "k8s.io/client-go/testing"
 )
 
+func TestDeletePVC(t *testing.T) {
+	tests := []struct {
+		name    string
+		pvcname string
+		wantErr bool
```

Comment:
One more use for failure as well

Reply:
Hi @surajnarwade ,

here in this test case, It is only validating whether the correct function is called with the correct parameter, So adding a failure case is not possible

Also according to the Unit test philosophy, we don't need to test the functionality of library function call eg: in this case `c.kubeClient.CoreV1().PersistentVolumeClaims(c.namespace).Delete(name, nil)`
It's out of our scope of testing because we are in an assumption that the library which we are using is tested by them, and we only need to validate the code which we have written.
This is what my understanding so far